models/mednist_gan/configs/metadata.json

```diff
@@ -1,20 +1,21 @@
 {
     "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_generator_20220718.json",
-    "version": "0.4.2",
+    "version": "0.4.3",
     "changelog": {
+        "0.4.3": "update to huggingface hosting and compatible with py3.10",
         "0.4.2": "add name tag",
         "0.4.1": "fix license Copyright error",
         "0.4.0": "update license files",
         "0.3.0": "Update for 1.0",
         "0.2.0": "Unify naming",
         "0.1.0": "Initial version"
     },
-    "monai_version": "1.0.0",
-    "pytorch_version": "1.10.0",
-    "numpy_version": "1.21.0",
+    "monai_version": "1.4.0",
```

Comment:
python3.10 (the version that our CI is used) does not support torch 1.10, thus I updated the dependencies for this bundle. cc @ericspod @KumoLiu